src/main/scala/com/lightbend/coursegentools/MasterSettings.scala

```diff
@@ -63,6 +63,8 @@ class MasterSettings(masterRepo: File, optConfigurationFile: Option[String]) {
 
   val relativeSourceFolder: String = config.getString("studentify.relative-source-folder")
 
+  val readmeFilesInExerciseRootFolder: Boolean = config.getBoolean("studentify.readme-files-in-exercise-root-folder")
```

Comment:
I'm not super-happy with the setting name. 
Maybe `readme-along-with-build-sbt` or `readme-in-root`, or define it the other way around like `readme-in-test-resources`?